description/urdf/sensor_mini58.urdf.xacro

```diff
@@ -1,14 +1,29 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
-       name="ati_mini58" >
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="ati_mini58" >
 
   <!-- ati_mini58 -->
-  <xacro:include filename="$(find ati_force_torque)/description/urdf/macros/mini58.urdf.xacro" />
+
```

Comment:
Please add here sim as input argument (see sr2_bringup macro)

Also add this argument into sensor.urdf.xacro and launch file

Can you also add parameter for sensor choice (mini_58 or ftm115)?